### PR TITLE
🎉 Add AnnotSV

### PR DESCRIPTION
**Pull request by dmiller15**
## Description

Merge after #131
Part of https://github.com/d3b-center/bixu-tracker/issues/1192

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## How Has This Been Tested?

- [x] https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/ed58772d-b15a-4e9a-9942-261b5a52e22d/ (runs successfully on a manta sv pass VCF)
- [x] https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/0f3808b9-4870-4736-b0fc-5ec3b605a810/ (empty outputs for when no SVs are found in the manta prepass VCF; still ran appropriately).
- [x] Rerunning with new passvcf input: https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/1b3fb6f5-e003-484a-a2e0-9c68d47ddd91/#

**Test Configuration**:
* Environment: cavatica
* Test files:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [x] I have committed any related changes to the PR

### CODE REVIEW

File: tools/annotsv.cwl [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+      - 'null'
+      - type: enum
+        name: annotaiton_mode
+        symbols: ["both","full","split"]
+    inputBinding:
+      prefix: "-annotationMode"
+      position: 1
+    doc: "Description of the types of lines produced by AnnotSV"
+#  bcftools: { type: 'string?', inputBinding: { prefix: "-bcftools", position: 1 }, doc: "Path of the bcftools local installation" }
+#  bedtools: { type: 'string?', inputBinding: { prefix: "-bedtools", position: 1 }, doc: "Path of the bedtools local installation" }
```
Comment (sickler-alex) [inline tools/annotsv.cwl:76]: These lines can get removed.

**Review by sickler-alex — APPROVED** [review]
Overall looks good, one small formatting issue.

**Review by bmennis — APPROVED** [review]
This looks good to me as well

File: tools/annotsv.cwl (hunk 1 above)
Comment (dmiller15) [reply tools/annotsv.cwl:76]: Done

**Review by migbro — APPROVED** [review]
Seems good. As you mentioned, we'll need to let ops and the scientist team know the tool behavior of when no SVs are present

**Pull request merged**